api/v1beta1/disruption_cron_webhook_test.go

```diff
@@ -121,6 +120,23 @@ var _ = Describe("DisruptionCron Webhook", func() {
 				})
 			})
 
+			When("disruptionTemplate is invalid", func() {
+				When("the count is invalid", func() {
```

Comment:
So I could add multiple tests and use DescribeTable for the various entries. But a valid DisruptionSpec is quite long, so it gets verbose. I could pass in a transformation function that takes a valid spec and breaks it in a specific way. But I feel the various cases for why a DisruptionSpec gets rejected are tested elsewhere. As long as we confirm we do validate the spec.disruptionTemplate of a DisruptionCron using DisruptionSpec's Validate, we can trust that all other cases are covered indirectly

Reply:
I'm ok with only testing like this as well.
Could we add a small comment there explaining that though?